counterpartylib/lib/messages/issuance.py

```diff
@@ -54,7 +55,14 @@ def initialise(db):
                       source_idx ON issuances (source)
                    ''')
 
-def validate (db, source, destination, asset, quantity, divisible, callable_, call_date, call_price, description, block_index):
+    # Add asset_longname for sub-assets
+    #   SQLite can’t do `ALTER TABLE IF COLUMN NOT EXISTS`.
+    columns = [column['name'] for column in cursor.execute('''PRAGMA table_info(issuances)''')]
+    if 'asset_longname' not in columns:
+        cursor.execute('''ALTER TABLE issuances ADD COLUMN asset_longname TEXT''')
+        cursor.execute('''CREATE UNIQUE INDEX IF NOT EXISTS asset_longname_idx ON issuances (asset_longname)''')
```

Comment:
tab this line one back, for consistency with the other CREATE INDEX lines (plus it will add the index back if the index is removed for some reason but the column name persists)